src/Config.ts

```diff
@@ -113,6 +116,20 @@ export class FacilitatorConfig {
     return new FacilitatorConfig({});
   }
 
+  /**
+   * Function reads the facilitator config from the specified path.
+   * If the file path does not exist empty configuration object is returned.
+   * @param filePath Path to facilitator config file.
+   * @returns Facilitator config object initialized by the specified file's content.
+   */
+  public static fromFile(filePath: string): FacilitatorConfig {
```

Comment:
I would recommend function to throw an exception if the specified file path does not exist.